src/scene/shader-lib/chunks/lit/frag/shadowPCSS.js

```diff
@@ -0,0 +1,185 @@
+export default /* glsl */`
+
+
+#define PCSS_SAMPLE_COUNT 16
+
+vec2 VogelDisk(float sampleIndex, float count, float phi)
+{
+    const float GoldenAngle = 2.4;
+    float r = sqrt(sampleIndex + 0.5) / sqrt(count);
+
+    float theta = sampleIndex * GoldenAngle + phi;
+
+    float sine, cosine;
+    sine = sin(theta);
+    cosine = cos(theta);
+
+    return vec2(r * cosine, r * sine);
+}
+
+vec3 VogelSphere(float sampleIndex, float count, float phi) {
+    float weight = sampleIndex / count;
+    float radius = sqrt(1.0 - weight * weight);
+    const float GoldenAngle = 2.4;
+
+    float theta = GoldenAngle * sampleIndex + phi;
+
+    return vec3(cos(theta) * radius, weight, sin(theta) * radius);
+}
+
+float GradientNoise(vec2 screenPos) {
```

Comment:
Do we have GLSL coding standard/conventions? Should function names be capitalized or camel-case? cc @mvaligursky

Reply:
I believe it's the same as Js code, so `gradientNoise`.

Reply:
Fixed

Reply:
is it fixed? It's still GradientNoise and not gradientNoise.